tree/tree/src/TLeaf.cxx

```diff
@@ -249,7 +249,8 @@ TString TLeaf::GetFullName() const
 TLeaf* TLeaf::GetLeafCounter(Int_t& countval) const
 {
    countval = 1;
-   const char* name = GetTitle();
+   auto slash = fTitle.First("/"); // for truncated types D32 F16
+   const char *name = (slash == TString::kNPOS) ? fTitle.Data() : TString(fTitle(0, slash)).Data();
```

Comment:
The 2nd part is returning 'deleting' memory (because the TString is a temporary). `name` probably needs to become a `TString`.